packages/server/src/init-repl-client.ts

```diff
@@ -1,9 +1,9 @@
-import { ReplClient } from "./repl";
+// import { ReplClient } from "./repl";
```

Comment:
This was calling things that eventually called files and code that doesn't exist anymore.